doc/release-notes.md

```diff
@@ -80,6 +80,24 @@ Low-level Changes section below.
 Low-level changes
 =================
 
+This section describes RPC changes mainly useful for testing, mostly not
+relevant in production. The changes are mentioned for completeness.
+
+RPC
+---
+
+- Example item
+
+Tests
+-----
+
+- The regression test chain, that can be enabled by the `-regtest` command line
+  flag, now requires transactions to not violoate standard policy by default.
```

Comment:
Should be "violate"? :-)

Reply:
Thx, will fix the typo the next time I touch the release notes or the next time I have to touch this pull (for other reasons)